index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JoseMontoya.com</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <header>
+        <div class="status_bar">
+            <p>9:41</p>
+            <img src="./assets/status_bar_icons.svg" alt="phone icons">
+        </div>
+        <div class="tool_bar">
+            <h1>Jose Montoya</h1>
+            <img src="./assets/menu_icon.svg" alt="menu">
+        </div>
+    </header>
+    <section class="headline">
+        <h2>Hey There. <br/> I'm Jose</h2>
+
+        <h3>I am a Software Developer</h3>
+
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.
+        </p>
+
+        <img class="social_media" src="./assets/social_media_icons.svg" alt="social media icons">
```

Comment:
- [ ] Great job using just one image for all icons However I think it would be better to change it. list  icon individually cause they will be linked to your social media 
The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
 
check this example below :+1: 
 ```
<ul>
                        <li><a href="#"><i class="fa fa-facebook fa-2x"></i></a></li>
                        <li><a href="#"><i class="fa fa-twitter fa-2x"></i></a></li>
                        <li><a href="#"><i class="fa fa-github fa-2x"></i></a></li>
                        <li><a href="#"><i class="fa fa-instagram fa-2x"></i></a></li>
                        <li><a href="#"><i class="fa fa-linkedin fa-2x"></i></a></li>
</ul>
```